src/foam/nanos/fs/File.js

```diff
@@ -86,25 +136,11 @@ foam.CLASS({
         });
       }
     },
-    {
-      class: 'String',
-      name: 'address',
-      label: 'Download Link',
-      transient: true,
-      expression: function (id) {
-        var sessionId = localStorage['defaultSession'];
```

Comment:
Fixed in https://github.com/kgrgreer/foam3/pull/625/commits/e70fb2deeb6eaa313fdf874e91505cbfe472d3fe.

Reply:
Why did we remove sessionId from address expression, so that we need to add it manually every time we want to get file address? like in e70fb2d filePreview?

Reply:
The address string field is a pointer to the file. The session id is only needed when we need to access the file itself. This  distinction enables the separation between the knowledge of the location of the file and the file access operation.

Reply:
Ok, it make sense. The last concern is that file Preview is not the only place where we use file.address. For instance InvoiceDetails, FileCard.